kore/src/Kore/Log/KoreLogOptions.hs

```diff
@@ -244,6 +248,32 @@ parseWarningSwitch =
         <> Options.help "Turn warnings into errors"
         )
 
+parseErrorEntries :: Parser EntryTypes
+parseErrorEntries =
+    option parseCommaSeparatedEntries info
+  where
+    info =
+        mempty
+        <> Options.long "error-entries"
+        <> Options.helpDoc ( Just nonErrorEntryTypes )
+
+    nonErrorEntryTypes :: OptPretty.Doc
+    nonErrorEntryTypes =
+        OptPretty.vsep
+            [ "Turn arbitrary log entries into errors"
+            , "Available entry types:"
+            , (OptPretty.indent 4 . OptPretty.vsep)
+                (OptPretty.text <$> filter notError getEntryTypesAsText)
+            {- The user can still give error entries as arguments, but it's
+                useless, so we don't list them here
+            -}
+            ]
+
+    notError :: String -> Bool
```

Comment:
I'm not really sure about this function, if the convention of how we name entry types ever changes we might forget to change this function as well. What do you think?

Reply:
Maybe we could split the registry from [Registry](https://github.com/kframework/kore/blob/master/kore/src/Kore/Log/Registry.hs) into a collection of types which are errors and those which are not. This way, we can use the types directly here. Does this make sense and do you think this would be better?

Reply:
I didn't split `registry` but `entryTypeReps` and `entryHelpDocs`. What do you think?